Stats improvements - Step 5: Handle corner cases design review

anitaa1990 opened the pull request:
This time PR fixes #1395 by modifying the color codes of the stats v4 availability banner and the stats v4 reverted banner. 

### Screenshots
<img width="300" src="https://user-images.githubusercontent.com/22608780/63915419-ade3e200-ca53-11e9-8bee-41c45062279a.png"> . <img width="300" src="https://user-images.githubusercontent.com/22608780/63915425-afada580-ca53-11e9-81ab-20cf847f93ba.png">

@kyleaparker could I trouble you for a review of this please! Thank you!!

Update release notes:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

anitaa1990 committed:
Changed color codes of stats availability and reverted banner text

kyleaparker reviewed (changes requested):
A few changes, 
- The spacing between the buttons and the text on the warning message (when stats is reverted) looks tighter than on the "try our improved stats" card. Can we increase the spacing to match?
- The text on the "try our improved stats" isn't aligned with the title. Is it possible to align this?
- There were colour changes also needed on the beta features screen in settings, will that be in a separate PR?

anitaa1990 committed:
Added padding to the availability banner subtitle to align with the title

anitaa1990 committed:
Added padding between the stats reverted banner subtitle and the buttons

anitaa1990 commented:
Thanks for the quick review @kyleaparker!

> The spacing between the buttons and the text on the warning message (when stats is reverted) looks tighter than on the "try our improved stats" card. Can we increase the spacing to match?

We sure can! I've increased the padding. The one of the LEFT is the before shot and the one of the RIGHT is the after shot:
 <img width="300" src="https://user-images.githubusercontent.com/22608780/63915425-afada580-ca53-11e9-81ab-20cf847f93ba.png"> . <img width="300" src="https://user-images.githubusercontent.com/22608780/64018924-6b053580-cb4b-11e9-9275-e94ecf26ec1d.png">

> The text on the "try our improved stats" isn't aligned with the title. Is it possible to align this?

Definitely! Adding screenshot below. The LEFT one is the before and the RIGHT is the after shot:
<img width="300" src="https://user-images.githubusercontent.com/22608780/63915419-ade3e200-ca53-11e9-8bee-41c45062279a.png"> . <img width="300" src="https://user-images.githubusercontent.com/22608780/64019122-e5ce5080-cb4b-11e9-82b1-823ab867a9dd.png">

> There were colour changes also needed on the beta features screen in settings, will that be in a separate PR?

Oops! Sorry, I completely missed that one! I've modified the subtitle text color and increased the line spacing between them. What do you think?

LEFT: before shot and RIGHT is after shot:
<img width="300" src="https://user-images.githubusercontent.com/22608780/64019207-1d3cfd00-cb4c-11e9-89b8-9cc0dac48d8a.png"> . <img width="300" src="https://user-images.githubusercontent.com/22608780/64019200-19a97600-cb4c-11e9-9cdd-853469d4354f.png">

anitaa1990 committed:
Added new attribute to the WCToggleSingleOptionView to display a separate text color for the subtitle

kyleaparker reviewed (changes requested):
Looks like this text is not quite aligned:

<img width="1059" alt="64019122-e5ce5080-cb4b-11e9-82b1-823ab867a9dd png (1080×2160) 2019-09-03 12-46-51" src="https://user-images.githubusercontent.com/712873/64192601-0bf43900-ce49-11e9-8ebf-c95ce66e8cda.png">

>Oops! Sorry, I completely missed that one! I've modified the subtitle text color and increased the line spacing between them. What do you think?

The spacing doesn't match what we are doing for the rest of the settings. Can we make sure this matches the other settings (ex. the space between Manage notifications and the subtext)

anitaa1990 committed:
Fixed padding between stats availability title and subtitle

anitaa1990 commented:
Thank you for the quick review @kyleaparker!!

> Looks like this text is not quite aligned:

My bad! Fixed below!
<img width="300" src="https://user-images.githubusercontent.com/22608780/64305449-1d565600-cfad-11e9-88a1-8aa2e1c092fb.png">

> The spacing doesn't match what we are doing for the rest of the settings. Can we make sure this matches the other settings (ex. the space between Manage notifications and the subtext)

This was because of the other settings items are single line and this one was multiple lines. Fixed below.

<img width="300" src="https://user-images.githubusercontent.com/22608780/64305753-f5b3bd80-cfad-11e9-9195-892959c86201.png">

anitaa1990 committed:
Added option to WcToggleSingleOptionView to add top padding

kyleaparker reviewed (approved):
👌

anitaa1990 commented:
Thank you @kyleaparker! @AmandaRiu @nbradbury @0nko, could one of you spare some time to take a look at this PR please? Thank you 🙇‍♀

nbradbury reviewed (approved):
Looks good! :shipit:
